### PR TITLE
Fixes #234 - Make sure the README is up to date.

### LINKED ISSUE #234: Ensure README is up to date
There's been some additions to the spec that may not be accurately reflected in the README.

### PR DESCRIPTION
**Pull request by miketaylr**
And other editorial fixes.

### CODE REVIEW

**Comment by miketaylr** [comment]
PTAL @amtunlimited, thanks!

File: README.md [hunk 1]
```diff
@@ -211,64 +269,101 @@ accomplish this as follows:
...
+User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko)
+            Chrome/71.0.0.0 Safari/537.36
+Sec-CH-UA: "Chrome"; v="74", ";Not)Your=Browser"; v="13"
+Sec-CH-UA-Mobile: ?0
+Sec-CH-UA-Platform: “Windows”
+Sec-CH-UA-Full-Version: “74.0.1234.5”
+```
+
 The user agent can make reasonable decisions about when to honor requests for detailed user agent
 hints and first-parties can use `Feature-Policy` to decide which third-parties they'd like to
```
Comment (amtunlimited) [inline README.md:344]: %:s/Feature-Policy/Permissions-Policy/g

File: README.md [hunk 2]
```diff
@@ -609,45 +672,72 @@ cases?
...
+## Aren’t we duplicating a lot of information already in the `User-Agent` header?
+
+It’s true that for some period of time there will be redundancy between the `User-Agent` header and
+the hints for platform, browser name and version, and mobileness. Over time, it may be possible to
+further freeze the User-Agent header to a single static string as the web platform adopts to Client
+Hints.
+
+## Aren’t you adding a lot of new headers? Isn’t that going to bloat requests?
+
+It’s true that we are adding up to 8 new headers per request. That’s quite a lot. But we don’t
```
Comment (amtunlimited) [inline README.md:719]: Maybe consider leaving the number out so we don't have to remember to change it if a new header gets added

**Review by amtunlimited — APPROVED** [review]
LGTM % nits

File: README.md (hunk 1 above)
Comment (miketaylr) [reply README.md:344]: Ah yes, nice catch!

File: README.md (hunk 2 above)
Comment (miketaylr) [reply README.md:719]: Yep, great feedback. Let's just say "multiple new headers".

**Pull request merged**